### ISSUE TITLE
Support CORS

### ISSUE THREAD

**Issue by ire4ever1190** [ire4ever1190/mike#50]
I have recently found a need for CORS and want utility functions to make life easier 

Might need to introduce browser testing like I have in tree to check I've correctly implemented it

Utils should be simple, maybe have a `manIHateCorsFunction` to handle 90% of use cases

**Comment by bwubta21122** [comment]
Yes you are right but really need the CORS functionality. Is there no way to use it now.... like right now?

**Comment by ire4ever1190** [comment]
> Is there no way to use it now.... like right now?

Can definitely be done without this PR. You just need to make sure to set [the needed headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#the_http_response_headers) when sending the responses

You can also do something like
```nim
"/^path" -> afterAny:
  # set all the headers in here
```
To mimic basically ignoring cors
